README.md

```diff
@@ -102,6 +102,7 @@ make deploy
 ### Current issues with the development
 We're currently in the process of transitioning into this web client from Scratch's existing structure. As we transition, there are going to be some issues along the way that relate to how this client needs to interact with the existing infrastructure to work properly in production.
 
+#### FALLBACK ####
```

Comment:
Humble nitpick, other headings in the README don't have the following `####`, i.e.:

```markdown
<!-- Bad -->
#### FALLBACK ####
#### Windows ####

<!-- Good -->
#### FALLBACK
#### Windows
```